semgrep/semgrep/core_runner.py

```diff
@@ -46,8 +47,8 @@
 
 logger = getLogger(__name__)
 
-RULE_SAVE_FILE = "semgrep_rules.yaml"
-TARGET_SAVE_FILE = "semgrep_targets.txt"
+RULE_SAVE_FILE = Path("~").expanduser() / USER_DATA_FOLDER / "semgrep_rules.yaml"
+TARGET_SAVE_FILE = Path("~").expanduser() / USER_DATA_FOLDER / "semgrep_targets.txt"
```

Comment:
```suggestion
RULE_SAVE_FILE = Path.home() / USER_DATA_FOLDER / "semgrep_rules.yaml"
TARGET_SAVE_FILE = Path.home() / USER_DATA_FOLDER / "semgrep_targets.txt"
```

Reply:
Leaving it for consistency with the other instance in the repo (and also my memory)